wpilibc/src/main/native/include/frc/smartdashboard/SendableChooser.h

```diff
@@ -46,8 +47,16 @@ class SendableChooser : public SendableChooserBase {
  public:
   ~SendableChooser() override = default;
 
-  void AddObject(wpi::StringRef name, T object);
-  void AddDefault(wpi::StringRef name, T object);
+  void AddOption(wpi::StringRef name, T object);
+  void SetDefaultOption(wpi::StringRef name, T object);
```

Comment:
Perhaps `AddOptionAsDefault()` would be a better name than `SetDefaultOption()`? It depends on whether this function always adds a new option or reuses one that's already created.

Reply:
It always adds a new option.  What is your thought on this naming @SamCarlberg ?

Reply:
I think this is fine as long as we document that the method also adds it as an option